### DISCUSSION TITLE
Can I use "drill by" feature?

### DISCUSSION THREAD

**Discussion by rindy8707** [apache/superset#23558] [Q&A / Help]
superset version: 2.0.1
I want to use the "drill by" feature. SO I write "DRILL_TO_DETAIL"=True to superset_config.py.
But, It is not Working. 

How can I use the "drill by" feature?

**Comment by mdeshmu** [comment]
Drill By is not released yet.

Drill to Detail is not same as Drill By. You can read more here: https://preset.io/blog/drill-down-and-drill-by/

**Reply by rindy8707** [reply]
Thank you for your response. 
I have reviewed the roadmap for the drill by feature. 
However, drill_to_detail functionality is still not activated. 
Is it true that drill_to_detail is possible in superset version 2.0.1?

**Reply by mdeshmu** [reply]
Drill to Detail is introduced in 2.1.0.

**Comment by mcoscielparamount** [comment]
Any update on when drill by will be released?   Thanks

**Reply by eschutho** [reply]
Drill by has been merged to master and will be released in 3.0. We're currently working on that release.

**Reply by Teesmo** [reply]
I am using 3.0.2 but don't see any Drill By feature? Has it been released? Do I have to do some configurations to enable it?

**Reply by mdeshmu** [reply]
its under DRILL_BY feature flag

**Reply by Teesmo** [reply]
Thank you

**Comment by amulya-mishra** [comment]
I'm currently testing the Drill By feature with pie charts in superset 3.0.0. I can drill by columns to see the chart as well as table view. However, it's only using the original aggregate metric to show details in drill by. For example, population count metric at state level can be drilled by city in the state, localities in the city, etc.

Is there any way or are there any plans to introduce dynamic metrics that can be changes while drilling by? For example, population count at state but average age at city level etc.

**Reply by madankgit** [reply]
Hi Amulya,

Please guide me on how the “Drill By” feature can be enabled. I would like to configure it so that drilling works on all columns instead of being limited to the original column.
